modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js

```diff
@@ -265,13 +265,12 @@ function runOperation (operationName) {
 //        payload = operationModule.generatePayload("TemperatureController", operationName, list["TemperatureController"]);
 //        serviceEndPoint = operationModule.getTemperatureControllerServiceEndpoint(operationName);
 //    }
-    console.log(payload);
     if(operationName == "NOTIFICATION"){
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158